aten/src/ATen/native/native_functions.yaml

```diff
@@ -1363,6 +1363,18 @@
     CPU: batch_norm_update_stats_cpu
     CUDA: batch_norm_update_stats_cuda
 
+- func: native_layer_norm(Tensor input, Tensor? weight, Tensor? bias, int[] normalized_shape, float eps) -> (Tensor, Tensor, Tensor)
+  dispatch:
+    CPU: LayerNormForwardCPU
+
+- func: native_layer_norm_backward(Tensor grad_out, Tensor input, Tensor mean, Tensor rstd, Tensor? weight, int[] normalized_shape, bool[3] output_mask) -> (Tensor, Tensor, Tensor)
+  dispatch:
+    CPU: LayerNormBackwardCPU
+
+- func: native_layer_norm_double_backward(Tensor? ggI, Tensor? ggW, Tensor? ggb, Tensor gO, Tensor input, Tensor mean, Tensor rstd, Tensor? weight, int[] normalized_shape, bool[3] output_mask) -> (Tensor, Tensor, Tensor)
```

Comment:
What does the "native_" prefix mean?

Reply:
Actually I am not quite sure about that, just follow the native_batch_norm function style. Is it better to remove it?